jxl/src/headers/frame_header.rs

```diff
@@ -407,6 +407,16 @@ pub struct FrameHeader {
 
     #[default(Extensions::default())]
     extensions: Extensions,
+
+    #[coder(Bits(0))]
+    #[default(nonserialized.img_width)]
```

Comment:
Let's initialize directly with the condition below, so that the functions become superflous